.cirrus.yml

```diff
@@ -280,18 +288,16 @@ testing_task:
     setup_environment_script: '$SCRIPT_BASE/setup_environment.sh |& ${TIMESTAMP}'
     unit_test_script: '$SCRIPT_BASE/unit_test.sh |& ${TIMESTAMP}'
     integration_test_script: '$SCRIPT_BASE/integration_test.sh |& ${TIMESTAMP}'
-    ginkgo_node_logs_script: 'cat $CIRRUS_WORKING_DIR/test/e2e/ginkgo-node-*.log || echo "Ginkgo node logs not found"'
-    df_script: '${DFCMD}'
-    audit_log_script: 'cat /var/log/audit/audit.log || cat /var/log/kern.log'
-    journalctl_b_script: 'journalctl -b'
 
     on_failure:
         failed_master_script: '$CIRRUS_WORKING_DIR/$SCRIPT_BASE/notice_master_failure.sh'
-        # Job has already failed, don't fail again and miss collecting data
-        failed_ginkgo_node_logs_script: 'cat $CIRRUS_WORKING_DIR/test/e2e/ginkgo-node-*.log || echo "Ginkgo node logs not found"'
-        failed_df_script: '${DFCMD}'
-        failed_audit_log_script: 'cat /var/log/audit/audit.log || cat /var/log/kern.log || echo "Uh oh, cat audit.log failed"'
-        failed_journalctl_b_script: 'journalctl -b || echo "Uh oh, journalctl -b failed"'
+
+    always:
+        # Carefuling - Don't fail again and miss collecting all data
+        ginkgo_node_logs_script: ${GINKGOLOGCMD}
+        df_script: ${DFCMD}
+        audit_log_script: ${AUDITLOGCMD}
+        journalctl_b_script: {JRNLLOGCMD}
```

Comment:
Missing dollar sign?

Reply:
Also, do you expect there ever to be a need for reusing the three new XXXCMDs? If not, might it be a lot simpler just to specify them as-is in the `always` block? First, it saves someone the time of having to scroll up and find them; second, it's much more readable and less error-prone than ABRVTDCMD (which, given the unreadable abbreviations, is easily subject to misspelling).

Reply:
oops.